package.json

```diff
@@ -27,9 +29,12 @@
     "react-dom": "~16.13.0",
     "react-native": "~0.63.3",
     "react-native-config": "^1.4.0",
+    "react-native-date-picker": "^3.2.5",
     "react-native-dotenv": "^2.4.1",
     "react-native-gesture-handler": "^1.7.0",
     "react-native-markdown-display": "^7.0.0-alpha.2",
+    "react-native-modalize": "^2.0.8",
+    "react-native-picker-select": "^8.0.4",
```

Comment:
Just double checking if we need all of these pickers.

Reply:
Removed `react-native-modalize`.